test/from-markdown/html/entities-char/output.yaml

```diff
@@ -0,0 +1,12 @@
+document:
+  data: {}
+  object: document
+  nodes:
+    - object: block
+      type: paragraph
+      nodes:
+        - object: text
+          leaves:
+            - object: leaf
+              marks: []
+              text: 'Hello world 🔌'
```

Comment:
Can you add a test for the "to-markdown" part?

Reply:
Done, But I've forgot to push